testdata/project-v3-with-deploy-image/controllers/busybox_controller.go

```diff
@@ -120,9 +164,26 @@ func (r *BusyboxReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		// the desired state on the cluster
 		return ctrl.Result{Requeue: true}, nil
 	}
+
 	return ctrl.Result{}, nil
 }
 
+// finalizeBusybox will perform the required operations before delete the CR.
+func (r *BusyboxReconciler) finalizeBusybox(log logr.Logger, cr *examplecomv1alpha1.Busybox) error {
+	// TODO(user): Add the cleanup steps that the operator
+	// needs to do before the CR can be deleted. Examples
+	// of finalizers include performing backups and deleting
+	// resources that are not owned by this CR, like a PVC.
+	// The following implementation will raise an event
+	r.recorder.Event(cr, "Normal", "Deleting",
```

Comment:
```suggestion
	r.recorder.Event(cr, "Warning", "Deleting",
```

Let's use WARNING so that we can get eaiser in the tests